packages/zoe/src/contractSupport/zoeHelpers.js

```diff
@@ -286,11 +333,58 @@ export const makeZoeHelpers = (zcf) => {
         details`issuer for ${keyword} must have natMathHelpers`,
       );
     },
+
     /**
      * Get the amountMath associated with a keyword
      * @param {Keyword} keyword
      */
     getAmountMath: keyword => zcf.getAmountMaths(harden([keyword]))[keyword],
+
+    /**
+     * Trade between left and right so that left and right end up with
+     * the declared gains.
```

Comment:
This isn't going to be compatible with facet parameters. 

Should we hold off on this change until facet parameters is in and figure out how to support facet params, or document it as incompatible (not to be used by contracts that allow keywords to vary across participants)? 

If this is submitted as is, I worry that I'm going to have to immediately rewrite it.

Reply:
Sure, I'm fine with holding off until after facet parameters are in.